.travis.yml

```diff
@@ -3,8 +3,13 @@ language: swift
 os:
   - osx
 
-osx_image: xcode10.3
-env: IPHONE_SIMULATOR_SDK='iphonesimulator12.4' OS_VERSION='12.4' IPHONE_NAME='iPhone Xs'
+matrix:
+  include:
+    - osx_image: xcode10.3
+      env: IPHONE_SIMULATOR_SDK='iphonesimulator12.4' OS_VERSION='12.4' IPHONE_NAME='iPhone Xs'
+    - osx_image: xcode11.5
```

Comment:
There is a new release of Xcode 11.6. I guess we want to use the latest version? I

Reply:
Changed to 11.6 and iOS 13.6